src/amo/components/Search/index.js

```diff
@@ -120,6 +124,18 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (!filters.query || filters.query.length === 0) {
```

Comment:
Shoot, I just realised this means a valid search without a query will error.

You should add a test for a search with `filters: { category: 'some-category', query: null }` and make sure it still renders results. That way we won't get caught out by this bug again. I guess really this should really just use `!hasSearchFilters(filters)` from `core/searchUtils`;